frontend/src/App.js

```diff
@@ -51,6 +51,12 @@ function App() {
     }
   }, [readyState]);
 
+  useEffect(() => {
+    if (lastJsonMessage) {
+        colorPixel(lastJsonMessage.position, lastJsonMessage.color);
```

Comment:
Use the other existing lastJsonMessage useEffect and parse different message types for colorPixel and nftMinted